### PR TITLE
Upgrade to 1.4.* branch

### PR DESCRIPTION
**Pull request by mplachta**
@ftrrtf

### CODE REVIEW

**Comment by filipgolonka** [comment]
:+1:

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 composer.lock
 vendor
-bin
\ No newline at end of file
+bin
+.idea
```
Comment (ftrrtf) [inline .gitignore]: @mplachta Could you remove `.idea` from ignore list? It's not related to bundle, I think it is better to add to the local gitignore.

Comment (mplachta) [inline .gitignore]: sure, just a sec

**Comment by mplachta** [comment]
@ftrrtf done with amend

**Comment by ftrrtf** [comment]
@mplachta Thank you for MR, unfortunately I do not have time to watch for js-updates right now.

**Pull request merged**